src/View.php

```diff
@@ -186,25 +192,102 @@ public function setDefaultParameters(array $defaultParameters): void
      *
      * @return void
      */
-    public function setBlocks(string $id, string $value): void
+    public function setBlock(string $id, string $value): void
     {
         $this->blocks[$id] = $value;
     }
 
     /**
      * {@see blocks}
      *
-     * @param string $value
+     * @param string $id
+     *
+     * @return void
+     */
+    public function unsetBlock(string $id): void
+    {
+        unset($this->blocks[$id]);
+    }
+
+    /**
+     * {@see blocks}
+     *
+     * @param string $id
      *
      * @return string
      */
-    public function getBlock(string $value): string
+    public function getBlock(string $id): string
     {
-        if (isset($this->blocks[$value])) {
-            return $this->blocks[$value];
+        if (isset($this->blocks[$id])) {
+            return $this->blocks[$id];
         }
 
-        throw new \InvalidArgumentException('Block: ' . $value . ' not found.');
+        throw new \InvalidArgumentException('Block: "' . $id . '" not found.');
+    }
+
+    /**
+     * {@see blocks}
+     *
+     * @param string $id
+     *
+     * @return bool
+     */
+    public function hasBlock(string $id): bool
+    {
+        return isset($this->blocks[$id]);
+    }
+
+    /**
+     * {@see data}
+     *
+     * @param string $id
+     * @param mixed $value
+     *
+     * @return void
+     */
+    public function setData(string $id, $value): void
+    {
+        $this->blocks[$id] = $value;
+    }
+
+    /**
+     * {@see data}
+     *
+     * @param string $id
+     *
+     * @return void
+     */
+    public function unsetData(string $id): void
```

Comment:
I think `remove` or `delete` is more readable and humanable. Could you rename please?

Reply:
https://github.com/yiisoft/arrays/blob/3fad714d9f8325fe9eb1bfac48d3fee70e68b860/src/ArrayHelper.php#L551

Thanks, fixed by analogy with ArrayHelper.